src/sections/Footer/Footer.tsx

```diff
@@ -24,143 +22,206 @@ const TopSection = styled.div`
   flex-direction: column;
+  justify-content: center;
   align-items: center;
+  margin-bottom: 1em;
+  border: 2px dashed white;
+  ${media.greaterThan('md')`
+    max-width: 20%
+  `}
+`;
+
+const ShowcaseCardText = styled.p`
+  text-align: center;
+  padding-top: 1.5em;
+  font-size: 16px;
+`;
+
+const SectionListItem = styled.li`
+  margin-bottom: 10;
+`;
+
+const commonText = css`
+  background-color: inherit;
+  border: none;
+  font-size: 16px;
+  font-weight: 300;
+  color: white;
+  &:hover {
+    color: black;
+    cursor: pointer;
+  }
 `;
 
-const menuData = [
+const SectionListButton = styled.button`
+  ${commonText}
+  padding: 0;
+`;
+
+const SectionListLink = styled.a`
+  ${commonText}
+
+  text-decoration: none !important;
+`;
+
+const Mail = styled.a`
+  ${commonText}
+
+  text-decoration: none !important;
+`;
+
+const siteMapData = [
+  {
+    name: 'Demo',
+    link: 'demo',
+  },
+  {
+    name: 'Docs',
+    link: 'docs',
+  },
+  {
+    name: 'Custom solutions',
+    link: 'https://www.vazco.eu/',
+  },
+];
+
+const firstStepsData = [
+  {
+    name: 'Installation',
+    link: 'https://forminer.com/docs/installation',
+  },
   {
-    title: 'Docs',
-    items: [
-      {
-        name: 'Introduction',
-        link: '/docs',
-      },
-      {
-        name: 'Supported technologies',
-        link: '/docs/supported-stack',
-      },
-      {
-        name: 'Integration',
-        link: '/docs/integration',
-      },
-    ],
+    name: 'Handling custom components',
+    link: 'https://forminer.com/docs/custom-components',
+  },
+  {
+    name: 'Integration',
+    link: 'https://forminer.com/docs/integration',
   },
 ];
 
 export const Footer = () => {
+  const theme = useTheme();
+  const columnView = useMediaQuery(theme.breakpoints.down('sm'));
+
+  const scrollToSection = (
+    event: MouseEvent<HTMLButtonElement>,
+    sectionId: string,
+  ) => {
+    event.preventDefault();
+    const pricingSection = document.getElementById(sectionId);
+
+    if (pricingSection) {
+      pricingSection.scrollIntoView();
+    }
+  };
+
   return (
     <StyledFooter>
       <Container>
         <TopSection>
-          <Box display="flex" flexWrap="wrap">
+          <Box
+            gridColumnGap={20}
+            display="flex"
+            flexDirection={columnView ? 'column' : 'row'}
+            alignContent="space-around"
+            justifyContent="center"
+            flexWrap="wrap"
+          >
+            <ShowcaseCardContainer>
+              <img
+                width={75}
+                height={75}
+                alt="forminer logo"
+                src="img/forminer-logo.svg"
+              />
+              <ShowcaseCardText>
+                Powerful tool for building forms in React
+              </ShowcaseCardText>
+            </ShowcaseCardContainer>
+            <StyledContainer>
+              <h5>Product</h5>
+              <ul>
+                <SectionListItem>
+                  <SectionListButton
```

Comment:
this button has different hover animation than links under, (that's because of transition)

https://github.com/vazco/forminer/assets/49162755/061b8f96-b779-4253-bfc8-4bb1a610cf71